token/js/src/instructions/associatedTokenAccount.ts

```diff
@@ -64,6 +65,31 @@ export function createAssociatedTokenAccountIdempotentInstruction(
     );
 }
 
+/**
+ * Construct a createAssociatedTokenAccountIdempotentInstructionShort instruction
```

Comment:
nit: let's keep the comment correct
```suggestion
 * Derive the associated token account and construct a CreateAssociatedTokenAccountIdempotent instruction
```